crates/wasm/src/lib.rs

```diff
@@ -11,3 +12,17 @@ pub fn compile(src: String) -> JsValue {
     let compiled_program = noirc_driver::Driver::compile_file(path, language);
     JsValue::from_serde(&compiled_program).unwrap()
 }
+// Deserialises bytes into ACIR structure
+#[wasm_bindgen]
+pub fn acir_from_bytes(bytes: Vec<u8>) -> JsValue {
+    console_error_panic_hook::set_once();
+    let circuit = Circuit::from_bytes(&bytes);
+    JsValue::from_serde(&circuit).unwrap()
```

Comment:
It seems some people suggest not to use JsValue::from_serde anymore?
https://github.com/rustwasm/wasm-bindgen/pull/3031